Document err_required field (#55)
tremor-rs/tremor-www-docs: docs/artefacts/onramps.md

The [`codec`](codecs.md) field is optional and if not provided will use onramps default codec.

> The `err_required` field can set to `true` if the source only should start once both `out` and `err` are connected to at least one pipeline.

Reviewer: ```suggestion
The `err_required` field can be set to `true` if the source should not start unless both `out` and `err` ports are connected to at least one pipeline.
```
